### DISCUSSION TITLE
Flight times, maximum distance

### DISCUSSION THREAD

**Discussion by alpha358** [StephenCarlson/MiniHawk-VTOL#38] [Q&A]
Hello, I have a few basic questions:

- What are the typical flight times with this UAV?
- What is the typical maximum distance it can travel from the operator (assuming unlimited radio range and the need to return)?
- What are the typical speeds that are achieved during fixed wing flight mode?

**Comment by StephenCarlson** [comment]
Hi @alpha358 !

- The typical hover-only time is about 2.5 to 3 minutes. In only forward-flight (hand-launching directly into flight, skipping hover), it gets around 9 to 10 minutes flight time in cruise.
- The aircraft cruises between 19 m/s and 21 m/s at half throttle, but can reach over 30 m/s at full throttle.
- In cruise at 20 m/s for about 8 minutes (with some reserve for landing), it can travel around 9km point-to-point, or 4.5 km out-and-back.

The metrics above are for the typical vehicle, with All-Up-Weight around 1200g, using a 160g 4s1300mAh LiPoly and the recommended 2550kV forward motors.

**Reply by alpha358** [reply]
Great, thank you for the reply!
